.github/workflows/build.yml

```
@@ -22,6 +22,12 @@ jobs:
             python: '3.9'
           - os: ubuntu-latest
             python: '3.10'
+          - os: ubuntu-latest
+            python: 'pypy-3.9'
+          - os: ubuntu-latest
+            python: 'pypy-3.8'
+          - os: ubuntu-latest
+            python: 'pypy-3.7'
```

Let's move the PyPy jobs to the top of the includes list; they're a bit slower to run so it's generally better to start the off sooner.

Also let's remove the hyphen, it's the more usual way of writing it:

```suggestion
          - os: ubuntu-latest
            python: 'pypy3.9'
          - os: ubuntu-latest
            python: 'pypy3.8'
          - os: ubuntu-latest
            python: 'pypy3.7'
```